### PR TITLE
Remove entry points impression event from CollectionsHubsNav

### PR DESCRIPTION
**Pull request by xtina-starr**
This removes the entry points impression event from `src/Components/v2/CollectionsHubsHomepageNav.tsx`. We remove it here because it fires too early and should be fired it from [force](https://github.com/artsy/force/pull/4784) upon the user scrolling within a specific area.

### CODE REVIEW

**Comment by xtina-starr** [comment]
I'm not sure if these generated files in this PR should be added. @pepopowitz @ashleyjelks thoughts?

**Comment by pepopowitz** [comment]
> I'm not sure if these generated files in this PR should be added. @pepopowitz @ashleyjelks thoughts?

Yeah, something seems up with that. I'm not sure I understand where those came from.

File: src/Components/v2/CollectionsHubsHomepageNav.tsx [hunk 1]
```diff
@@ -18,7 +18,6 @@ export const CollectionsHubsHomepageNav = track(
     context_page: AnalyticsSchema.PageName.HomePage,
     context_module: AnalyticsSchema.ContextModule.CollectionHubEntryPoint,
     subject: AnalyticsSchema.Subject.FeaturedCategories,
-    action_type: AnalyticsSchema.ActionType.Impression,
```
Comment (pepopowitz) [inline src/Components/v2/CollectionsHubsHomepageNav.tsx]: Is this change removing the actual impression event, or just removing metadata and the event is still firing? (It's possible that there's a second file that changed, I'm not seeing it right now with the relay noise though.)

Comment (ashleyjelks) [reply src/Components/v2/CollectionsHubsHomepageNav.tsx]: Plus one that we can remove these generated files from the PR. I was looking at the PR in Force and I'm wondering why both of these tracking events are needed, both here in Reaction and there in Force?

**Comment by xtina-starr** [comment]
Closing this in favor of https://github.com/artsy/reaction/pull/3025.

**Pull request closed without merging**